app/components/chat-text-options.hbs

```diff
@@ -0,0 +1,10 @@
+<div id="chat-text-options" class="cool-button">
+  <div title={{t "chat.color"}}>
+    <input 
+      type="color"
+      value="#fff940"
+      oninput={{action "selectedColor"}}
```

Comment:
I think the new syntax should be
`on "input" this.selectedColor`

https://guides.emberjs.com/release/components/component-state-and-actions/#toc_html-modifiers-and-actions

Reply:
Thanks for the heads up! Made the change here https://github.com/datafruits/datafruits/commit/26f6e3a81e8cd027590c7f891ce8a29de479ecd1#diff-bf574a10216f3603e6aec60c00f102d3R6